drivers/i2c/i2c_ll_stm32_v2.c

```diff
@@ -1248,6 +1461,10 @@ int i2c_stm32_transaction(const struct device *dev,
 			  struct i2c_msg msg, uint8_t *next_msg_flags,
 			  uint16_t periph)
 {
+	int ret = 0;
```

Comment:
prefer with an empty line betwen local variable definitions and 1st instructions.